fusionfrontend/src/app/factory/components/pages/assets-grid-page/assets-grid-page.component.ts

```diff
@@ -46,8 +48,20 @@ export class AssetsGridPageComponent implements OnInit, OnDestroy {
     this.rooms$ = this.factoryResolver.rooms$;
     this.assets$ = this.factoryResolver.assets$;
     this.assetsWithFields$ = this.factoryResolver.assetsWithFields$;
+
+    this.assetsWithFields$.subscribe(assetsWithFields => this.updateCommonFields(assetsWithFields));
   }
 
   ngOnDestroy() {
   }
+
+  private updateCommonFields(assetsWithFields: AssetWithFields[]): void {
+    this.commonFields = [];
+    if (assetsWithFields && assetsWithFields.length > 0) {
+      this.commonFields = assetsWithFields[0].fields;
+      assetsWithFields.forEach(assetWithFields => {
+        this.commonFields = this.commonFields.filter(o => assetWithFields.fields.some(({ description}) => o.description === description));
```

Comment:
Du könntest das "o" noch in etwas lesbareres ändern

Reply:
wohl wahr ;)